src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,24 @@
+import { HStack, Flex } from "@chakra-ui/react"
+
+export interface FooterProps {
+  children: JSX.Element
+}
+
+export const Footer = ({ children }: FooterProps): JSX.Element => (
+  <Flex
+    h="80px"
+    w="100%"
+    pr="30px"
+    position="fixed"
+    background="background.action.defaultInverse"
+    border="1px"
+    borderColor="border.divider.alt"
+    justify="end"
+    align="center"
+    bottom="0"
+  >
+    <HStack>
```

Comment:
just double checking here - is the intention for the children of `HStack` to be treated as a singular unit and justified to the end of the footer with 1rem spacing? 

this is the current behaviour and if design shows that the footer is consistent liek this across all pages then it's fine